api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/EriDSDDataset.java

```diff
@@ -526,14 +526,92 @@ public DataSetDefinition constructEriDSDDataset() {
             mappings),
         "");
 
+    dsd.addColumn(
+        "N5T",
+        "DSD N5 Total",
+        EptsReportUtils.map(
+            eptsGeneralIndicator.getIndicator(
+                "N5T",
+                EptsReportUtils.map(
+                    eriDSDCohortQueries.getPatientsWhoAreActiveAndParticipatingInAccessionClubs(),
+                    mappings)),
+            mappings),
+        "");
+    dsd.addColumn(
+        "N5SST",
+        "DSD N5 Stable subtotal",
+        EptsReportUtils.map(
+            eptsGeneralIndicator.getIndicator(
+                "N5SST",
+                EptsReportUtils.map(
+                    eriDSDCohortQueries
+                        .getPatientsWhoAreActiveParticipatingInAccessionClubsAndStable(),
+                    mappings)),
+            mappings),
+        "");
+    dsd.addColumn(
+        "N5SNPNBA",
+        "DSD N5 Stable Non-pregnant and Non-Breastfeeding Adults (>=15)",
+        EptsReportUtils.map(
+            eptsGeneralIndicator.getIndicator(
+                "N5SNPNBA",
+                EptsReportUtils.map(
+                    eriDSDCohortQueries.getN5StableNonPregnantNonBreastfeeding(), mappings)),
```

Comment:
If you address the first comment. then you can easily replace this method call with "eriDSDCohortQueries.getPatientsWhoAreActiveParticipatingInAccessionClubsAndStable()"